rmgpy/tools/loader.py

```diff
@@ -180,4 +189,3 @@ def loadRMGJavaJob(inputFile, chemkinFile=None, speciesDict=None, useChemkinName
         species.molecule[0].draw(str(path))
     
     return rmg
-
```

Comment:
Can you remove the white space changes here?  These tend to clog our commit log and make it more difficult to determine what changes were made in a given commit.